recipe/meta.yaml

```diff
@@ -25,9 +25,8 @@ requirements:
     - proj
     - gdal
     - xorg-libx11
-    - xorg-xproto
-    - libopengl
-    - libgl-devel
+    - xorg-xorgproto
```

Comment:
Please explain why you had to replace `xorg-xproto`, `libopengl`, and `libgl-devel` with `xorg-xorgproto` and `mesalib`.